recipes/jimtcl/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "jimtcl" %}
+{% set version = "0.82" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/msteveb/jimtcl/archive/refs/tags/{{ version }}.tar.gz
+    sha256: e8af929b815e4d30e54ff116b2b933e56c00a02b9110529d1a58660b2469aea7
+  - path: build.sh  # [win]
```

Comment:
This is part of the recipe.
```suggestion
```

Reply:
I included this because I was finding that windows builds only copied the `bld.bat`, not the `build.sh`, and since the Windows builds use Mingw32 I wrote `bld.bat` to call `build.sh` — is it not necessary in that case?

Reply:
It should also copy the `build.sh` we use that in several other packages without the need to include it here.